### PR TITLE
Split articles and drafts in blog and order by date

### PR DESCRIPTION
**Pull request by FrancoisDoussin**
## Type
Feature

## Purpose
This PR order Articles by createdAt date.

## BC Break
NO

### CODE REVIEW

File: Bundle/BlogBundle/Repository/ArticleRepository.php [hunk 1]
```diff
@@ -143,4 +143,23 @@ public function getVeryLastRecord()
...
+     *
+     * @return array()
+     */
+    public function getByBlogOrderBy(Blog $blog, $order)
+    {
+        $queryBuilder = $this->getAll(false, $blog)
+            ->getInstance();
+
+        return $queryBuilder
+            ->orderBy('article.createdAt', $order)
```
Comment (alexislefebvre) [inline Bundle/BlogBundle/Repository/ArticleRepository.php]: I think that it should be `publishedAt`?

**Comment by alexislefebvre** [comment]
See #972.

**Comment by alexislefebvre** [comment]
I'm voting in favour of creating two lists, I find `UNION` requests ugly. :upside_down_face:

File: Bundle/BlogBundle/Resources/views/Blog/Tabs/_drafts.html.twig [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{% trans_default_domain "victoire" %}
```
Comment (lenybernard) [inline Bundle/BlogBundle/Resources/views/Blog/Tabs/_drafts.html.twig]: don't you think you should extend the `Blog:Tabs/_articles.html.twig` template ?

File: Bundle/BlogBundle/Resources/translations/victoire.en.xliff [hunk 3]
```diff
@@ -2,259 +2,259 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
   <file date="2013-11-28T17:11:45Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
     <body>
-      <trans-unit id="58893b9921bc7" resname="form.widget.tab.article.label">
+      <trans-unit id="5ae71b95ccfef" resname="form.widget.tab.article.label">
```
Comment (lenybernard) [inline Bundle/BlogBundle/Resources/translations/victoire.en.xliff]: please don't change ids when it's not needed

File: Bundle/BlogBundle/Repository/ArticleRepository.php [hunk 4]
```diff
@@ -143,4 +143,48 @@ public function getVeryLastRecord()
             ->getQuery()
             ->getOneOrNullResult();
     }
+
+    /**
+     * Get Article by blog orderBy DESC.
```
Comment (lenybernard) [inline Bundle/BlogBundle/Repository/ArticleRepository.php]:
```diff
-Get Article by blog orderBy DESC.
+Get articles by blog ordered by publication DESC.
```

**Comment by lenybernard** [comment]
Can you please do the same for the 2.3 branch ? 😘

**Pull request merged**